src/main/java/com/epam/prejap/teatrees/block/PBlock.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.prejap.teatrees.block;
+/**
+ * This class introduces the P-shaped block in a Tetris game.
+ * @author Bartłomiej Słomiński
```

Comment:
```suggestion
 * This class introduces the P-shaped block in a Tetris game.
 *
 * @author Bartłomiej Słomiński
```

Yet `@author` should demlimeted from the class overview